jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsPresenter.java

```diff
@@ -164,6 +164,7 @@ public void setIsForLog(boolean isForLog) {
     }
 
     public void onProcessSelectionEvent(@Observes ProcessInstanceSelectionEvent event) {
+        boolean refreshDetails = (event != null && event.getProcessInstanceId() != null && event.getProcessInstanceId().equals(processInstanceId));
```

Comment:
This can be shortened as:
```java
boolean refreshDetails = (event != null && Objects.equals(event.getProcessInstanceId(), processInstanceId));
```